ethereum/event/BREventQueue.c

```diff
@@ -82,6 +97,7 @@ eventQueueClear (BREventQueue queue) {
     queue->pending = NULL;
     queue->available = NULL;
 
+    pthread_cond_destroy(&queue->cond);
```

Comment:
I don't think we want to destroy this here, but rather in `eventQueueDestroy`?

Reply:
Fixed.